frame/evm/src/runner/stack.rs

```diff
@@ -62,6 +77,8 @@ where
 		config: &'config evm::Config,
 		precompiles: &'precompiles T::PrecompilesType,
 		is_transactional: bool,
+		weight_limit: Option<Weight>,
+		transaction_len: Option<u64>,
```

Comment:
Suggest renaming to something like `proof_size_used`. This value is not merely the transaction length.

Reply:
Although this _is_ indeed the encoded extrinsic length, I agree the value is purely used proof size accounting and also we might want to account for some additional base cost in the future. I'm renaming it to `proof_size_base_cost`.